### ISSUE TITLE
Incomplete man page

### ISSUE THREAD

**Issue by jmgibson1981** [elementary/terminal#728]
### What Happened?

In https://github.com/elementary/terminal/issues/395 there was added an option using the -x flag instead of -e. I have not looked at the code but it was mentioned that it was done to match gnome-terminal.

This fixed the issue of not being able to utilize the terminal in CodeBlocks properly. However the man pages have not been updated to match this. Current man page OPTIONS list.

```
-h, --help
              Show help message and exit

       -v, --version
              Print version info and exit

       -e, --execute=PATH
              Execute the specified binary with given parameters in terminal.

       -w, --working-directory=PATH
              Set the specified directory as shell working directory on startup
```

### Steps to Reproduce

1. command - man io.elementary.terminal

### Expected Behavior

Expected to see fix from 395 documented.

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
It doesn't fail. This is a documentation bug.
```

**Comment by jmgibson1981** [comment]
I didn't know where to find the man page sources. Patch for the man page below. Information extracted from the Application.vala. This also adds the other options found there that aren't in the current man page.

```
--- io.elementary.terminal.1	2023-05-31 05:19:04.704833428 -0700
+++ manpageupdate	2023-05-31 05:23:42.370667837 -0700
@@ -13,11 +13,23 @@
 .BR \-h ", " \-\-help
 Show help message and exit
 .TP
+.BR \-n ", " \-\-help
+Forces a new window
+.TP
+.BR \-h ", " \-\-help
+Forces a new tab
+.TP
+.BR \-m ", " \-\-help
+Starts terminal in a minimized state
+.TP
 .BR \-v ", " \-\-version
 Print version info and exit
 .TP
 .BR \-e ", " \-\-execute =\fIPATH\fR
-Execute the specified binary with given parameters in terminal.
+Execute the specified binary with given parameters in terminal
+.TP
+.BR \-x ", " \-\-execute =\fIPATH\fR
+Is used for using the rest of the command line in the new tab/window
 .TP
 .BR \-w ", " \-\-working-directory =\fIPATH\fR
 Set the specified directory as shell working directory on startup
```

**Comment by ryonakano** [comment]
Seems like this was fixed in #730 so closing as completed.
